nemo_skills/prompt/few_shot_examples/minif2f.py

```diff
@@ -0,0 +1,475 @@
+minif2f_deepseek_fewshot = [
+    {
+        "name": "mathd_algebra_182",
```

Comment:
Remove all keys that are not necessary. Only keep what's referenced inside fewshots examples section of your config

Reply:
Maybe for models that are not trained for lean4 specifically we would need all the fewshots? like for 405b? 

Maybe I could just give 5 examples by default? 
```
examples_map = {
    "minif2f_deepseek_fewshot": minif2f_deepseek_fewshot[:5],
}
```